pkg/crc/machine/start.go

```diff
@@ -50,50 +51,45 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 			return startError(startConfig.Name, "Failed to ask for pull secret", err)
 		}
 
-		machineConfig := config.MachineConfig{
-			Name:       startConfig.Name,
-			BundleName: filepath.Base(startConfig.BundlePath),
-			CPUs:       startConfig.CPUs,
-			Memory:     startConfig.Memory,
-		}
-
-		crcBundleMetadata, err = getCrcBundleInfo(startConfig.BundlePath)
+		bundle, err = getCrcBundleInfo(startConfig.BundlePath)
 		if err != nil {
 			return startError(startConfig.Name, "Error getting bundle metadata", err)
 		}
 
-		logging.Infof("Checking size of the disk image %s ...", crcBundleMetadata.GetDiskImagePath())
-		if err := crcBundleMetadata.CheckDiskImageSize(); err != nil {
-			return startError(startConfig.Name, fmt.Sprintf("Invalid bundle disk image '%s'", crcBundleMetadata.GetDiskImagePath()), err)
+		logging.Infof("Checking size of the disk image %s ...", bundle.GetDiskImagePath())
+		if err := bundle.CheckDiskImageSize(); err != nil {
+			return startError(startConfig.Name, fmt.Sprintf("Invalid bundle disk image '%s'", bundle.GetDiskImagePath()), err)
 		}
 
-		openshiftVersion := crcBundleMetadata.GetOpenshiftVersion()
+		openshiftVersion := bundle.GetOpenshiftVersion()
 		if openshiftVersion == "" {
 			logging.Info("Creating VM...")
 		} else {
 			logging.Infof("Creating CodeReady Containers VM for OpenShift %s...", openshiftVersion)
 		}
 
-		// Retrieve metadata info
-		diskPath := crcBundleMetadata.GetDiskImagePath()
-		machineConfig.DiskPathURL = fmt.Sprintf("file://%s", filepath.ToSlash(diskPath))
-		machineConfig.SSHKeyPath = crcBundleMetadata.GetSSHKeyPath()
-		machineConfig.KernelCmdLine = crcBundleMetadata.Nodes[0].KernelCmdLine
-		machineConfig.Initramfs = crcBundleMetadata.GetInitramfsPath()
-		machineConfig.Kernel = crcBundleMetadata.GetKernelPath()
-
-		_, err := createHost(libMachineAPIClient, machineConfig)
+		_, err := createHost(libMachineClient, config.MachineConfig{
```

Comment:
slight preference for this as it's easier to parse:
```
machineConfig := config.MachineConfig {
...
}
_, err := createHost(libMachineClient, machineConfig)
```
(nit: it's neither a renaming nor related to `IsRunning` -> this could actually be done in `Extract functions that return large structs`)

Reply:
IsRunning is gone, removed by an other commit.